joomla/joomla-cms
# [6.2] feat(task/checkfiles): Introduce image compression

### Summary of Changes

Introduces image compression to the image resizer plugin.

> Why?

Resizing might result in a size increase, so we leave an option to compress.


### Testing Instructions

- Drop some images in a media folder (mix PNG, WEBP, JPG)
- Configure a checkfiles task to resize/compress these images
- Verify that the resized images are compressed with the right level.

### Actual result BEFORE applying this Pull Request

No compression option. Image size may go up after the resize process.



### Expected result AFTER applying this Pull Request

Compression option.


### Link to documentations
Please select:
